django_prices/templatetags/prices_i18n.py

```diff
@@ -66,22 +66,15 @@ def format_price(value, currency, html=False, normalize=False):
     return mark_safe(result)
 
 
-@register.simple_tag
-def gross(price, html=False, normalize=False):
-    if html or normalize:
-        return format_price(price.gross, price.currency, html, normalize)
-    return currencyfmt(price.gross, price.currency)
-
-
-@register.simple_tag
-def net(price, html=False, normalize=False):
-    if html or normalize:
-        return format_price(price.net, price.currency, html, normalize)
-    return currencyfmt(price.net, price.currency)
-
-
-@register.simple_tag
-def tax(price, html=False, normalize=False):
-    if html or normalize:
-        return format_price(price.tax, price.currency, html, normalize)
-    return currencyfmt(price.tax, price.currency)
+@register.filter
+def amount(obj, normalize=False):
```

Comment:
Wouldn't it make more sense if we had a separate tag that does normalization and used the parameter of `amount` to specify the format (text vs html)?

Reply:
The first version is definitely much more readable.

Reply:
Yes I was thinking about this and wanted to discuss this kind of pipeline. Now there is nothing to discuss :)